main/nfs-utils/APKBUILD

```diff
@@ -14,12 +14,12 @@ makedepends="util-linux-dev libtirpc-dev libcap-dev libevent-dev
 	libnfsidmap-dev keyutils-dev lvm2-dev krb5-dev sqlite-dev
 	autoconf automake libtool bsd-compat-headers"
 subpackages="$pkgname-doc $pkgname-dbg rpcgen"
-
-source="http://downloads.sourceforge.net/nfs/$pkgname-$_basever.tar.bz2
+source="$pkgname-$pkgname-$_pkgver.tar.gz::http://git.linux-nfs.org/?p=steved/nfs-utils.git;h=refs/tags/nfs-utils-$_pkgver;a=snapshot;sf=tgz
```

Comment:
you it be an idea to use this instead of gitweb? https://www.kernel.org/pub/linux/utils/nfs-utils/

Reply:
Yes, if I had known it existed xD

Reply:
Third time's the charm... Switched to kernel.org.